File: ClientesOrionTek/app/src/main/java/com/example/clientesoriontek/data/remote/model/AddressDto.kt
package com.example.clientesoriontek.data.remote.model

data class AddressDto(
    val addressId: Long = 0,
    val clientId: Long,
    val street: String,
    val city: String,
    val sector: String,
    val houseNumber: String
)

data class CreateAddressDto(
    val clientId: Long,
    val street: String,
    val city: String,
    val sector: String,
    val houseNumber: String
)

data class UpdateAddressDto(
    val street: String,
    val city: String,
    val sector: String,
    val houseNumber: String
)

File: ClientesOrionTekBackEnd/DTOs/AddressDto.cs
namespace ClientesOrionTekBackEnd.DTOs;

public class AddressDto
{
    public long AddressId { get; set; }
    public long ClientId { get; set; }
    public string Street { get; set; } = string.Empty;
    public string City { get; set; } = string.Empty;
    public string Sector { get; set; } = string.Empty;
    public string HouseNumber { get; set; } = string.Empty;
}

public class CreateAddressDto
{
    public long ClientId { get; set; }
    public string Street { get; set; } = string.Empty;
    public string City { get; set; } = string.Empty;
    public string Sector { get; set; } = string.Empty;
    public string HouseNumber { get; set; } = string.Empty;
}

public class UpdateAddressDto
{
    public string Street { get; set; } = string.Empty;
    public string City { get; set; } = string.Empty;
    public string Sector { get; set; } = string.Empty;
    public string HouseNumber { get; set; } = string.Empty;
}
